plugins/libplugin-pay.c

```diff
@@ -791,7 +806,7 @@ static bool payment_route_check(const struct gossmap *gossmap,
 				struct amount_msat amount,
 				struct payment *p)
 {
-	struct short_channel_id scid;
+	struct short_channel_id_dir scid;
```

Comment:
The type changed, maybe we should also change the name: `scid` is usually used for an instance of `short_channel_id` while `scidd` indicates it is a `short_channel_id_dir`.

Reply:
Done, thanks